scanpipe/pipes/resolve.py

```diff
@@ -45,8 +46,10 @@
 def resolve_packages(input_location):
     """Resolve the packages from manifest file."""
     default_package_type = get_default_package_type(input_location)
+    # if resolving packages from a codebase/package we do nothing if a
+    # file at input_location is not a package manifest.
```

Comment:
Changed this thanks.

I added this to explain why we are not failing here anymore in contrast to before, but that should not be with the code, explaining here is sufficient. This can be removed too if that's better